### PR TITLE
[now-cli] Fix broken message when a deployment is canceled

### PR DESCRIPTION
**Pull request by anatrajkovska**
This PR fixes an issue with displaying the message when a deployment is canceled, either when it is canceled from the UI, or when a deployment is automatically canceled if no change has been made in the project.

### CODE REVIEW

**Review by AndyBitz — APPROVED** [review]

File: packages/now-cli/src/commands/deploy/latest.js [hunk 1]
```diff
@@ -576,7 +576,7 @@ export default async function main(
     }
 
     if (deployment.readyState === 'CANCELED') {
-      output.log('The deployment has been canceled.');
+      output.print('The deployment has been canceled.');
```
Comment (TooTallNate) [inline packages/now-cli/src/commands/deploy/latest.js]: Why this change? It will remove the trailing newline, which I think we don't want to happen.

**Review by AndyBitz — APPROVED** [review]

**Review by AndyBitz — APPROVED** [review]

File: packages/now-cli/src/commands/deploy/latest.js (hunk 1 above)
Comment (anatrajkovska) [reply packages/now-cli/src/commands/deploy/latest.js]: @TooTallNate Yes you are right, I just fixed this by adding a newline at the end, please check.
